sdk/contentunderstanding/Azure.AI.ContentUnderstanding/README.md

```diff
@@ -0,0 +1,260 @@
+
+> **Note:** The configuration is persisted in your Microsoft Foundry resource, so you only need to run this once per resource (or whenever you change your deployment names). If you have multiple Microsoft Foundry resources, you need to configure each one separately.
+
+### Authenticate the client
+
+To authenticate the client, you need your Microsoft Foundry resource endpoint and credentials. You can use either an API key or Azure Active Directory (Azure AD) authentication.
+
+#### Using DefaultAzureCredential
+
+The simplest way to authenticate is using `DefaultAzureCredential`, which supports multiple authentication methods and works well in both local development and production environments:
+
+```C# Snippet:CreateContentUnderstandingClient
+string endpoint = "<endpoint>";
+var credential = new DefaultAzureCredential();
+var client = new ContentUnderstandingClient(new Uri(endpoint), credential);
+```
+
+#### Using API Key
+
+You can also authenticate using an API key from your Microsoft Foundry resource:
+
+```C# Snippet:CreateContentUnderstandingClientApiKey
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var client = new ContentUnderstandingClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
+
+To get your API key:
+1. Go to [Azure Portal][azure_portal]
+2. Navigate to your Microsoft Foundry resource
+3. Go to **Resource Management** > **Keys and Endpoint**
+4. Copy one of the **Keys** (Key1 or Key2)
+
+For more information on authentication, see [Azure Identity client library][azure_identity_readme].
+
+## Key concepts
+
+### Prebuilt Analyzers
+
+Content Understanding provides prebuilt analyzers that are ready to use without any configuration. These analyzers use the `*Search` naming pattern:
+
+* **`prebuilt-documentSearch`** - Extracts content from documents (PDF, images, Office documents) with layout preservation, table detection, figure analysis, and structured markdown output. Optimized for RAG scenarios.
+* **`prebuilt-audioSearch`** - Transcribes audio content with speaker diarization, timing information, and conversation summaries. Supports multilingual transcription.
+* **`prebuilt-videoSearch`** - Analyzes video content with visual frame extraction, audio transcription, and structured summaries. Provides temporal alignment of visual and audio content.
+
+> **Note:** The prebuilt analyzers use the `prebuilt-{type}Search` naming pattern (not `prebuilt-{type}Analyzer`). This is a recent change in the Content Understanding service.
+
+### Content Types
+
+The API returns different content types based on the input:
+
+* **`document`** - For document files (PDF, images, Office documents). Contains pages, tables, figures, paragraphs, and markdown representation.
```

Comment:
From an SDK level, perhaps we should use DocumentContent and AudioVideoContent instead of describing the enum value.